doc/classes/ArrayMesh.xml

```diff
@@ -165,6 +165,13 @@
 				Returns the primitive type of the requested surface (see [method add_surface_from_arrays]).
 			</description>
 		</method>
+		<method name="surface_remove">
+			<return type="void" />
+			<param index="0" name="surf_idx" type="int" />
+			<description>
+				Removes a surface at position surf_idx, shifting greater surfaces one surf_idx slot down.
```

Comment:
@clayjohn @Repiteo These changes should really not have been dismissed just because there was disagreement over codestyle changes, please pay attention to all suggested changes and don't skip some ones by relevant teams